.gitmodules

```diff
@@ -0,0 +1,5 @@
+[submodule "ADLX"]
+	path = vendor/ADLX
+	url = https://github.com/GPUOpen-LibrariesAndSDKs/ADLX
+	depth = 1
+	update = none
```

Comment:
Since this repo is quite large while we only need one folder from it, we might look into something like https://git-scm.com/docs/git-sparse-checkout except that it's experimental.